### PR TITLE
fix: add jira integration to the SDK

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

This PR adds jira automation to automatically create jira for every GitHub issue we have.

EDIT:

Need to figure out secret values.

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: .github/workflows/issues.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                      "id": "19383"
+                  },
+                  "summary": "HELP: '"${ISSUE_TITLE_NO_SPECIAL_CHARS}"'",
+                  "issuetype": {
+                      "id": "3"
+                  },
+                  "description": "This ticket tracks the following GitHub issue: '"${ISSUE_URL}"'.",
+                  "components": [
+                      {
+                          "id": "27590"
```
Comment (andreaangiolillo) [inline .github/workflows/issues.yml:45]: make sure the component id is the one for CLOUDP

Comment (wtrocki) [reply .github/workflows/issues.yml:45]: I shamelessly copied your work from Terraform without attribution. 
We need to figure out secrets we need for that job

File: .github/workflows/issues.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          ISSUE_TITLE_NO_SPECIAL_CHARS=$(echo "$ISSUE_TITLE" | tr -d '!$&*@#"\047\140')
+          json_response=$(curl --request POST \
+            --url 'https://jira.mongodb.org/rest/api/2/issue' \
+            --header 'Authorization: Bearer '"${JIRA_API_TOKEN}" \
+            --header 'Accept: application/json' \
+            --header 'Content-Type: application/json' \
+            --data '{
+              "fields": {
+                  "project": {
+                      "id": "19383"
```
Comment (andreaangiolillo) [inline .github/workflows/issues.yml:36]: this project is INTMDB, you need to get the id for CLOUDP

Comment (andreaangiolillo) [reply .github/workflows/issues.yml:36]: The cloudp ID is `10984`
```suggestion
                      "id": "10984"
```

**Pull request merged**